model/project_test.go

```diff
@@ -824,6 +830,33 @@ func (s *projectSuite) TestAliasResolution() {
 	s.Empty(displayTaskPairs)
 }
 
+func (s *projectSuite) TestCheckRunCount() {
+	pairs := TaskVariantPairs{
+		ExecTasks: TVPairSet{
+			TVPair{
```

Comment:
can we add some tasks that arent a check run just so we know the test is accurate

Reply:
done